source/apiVolontaria/volunteer/models.py

```diff
@@ -199,3 +193,32 @@ def is_expired(self):
     @property
     def is_active(self):
         return self.cycle.is_active
+
+
+class Participation(models.Model):
+    """This class represents the Participation model."""
```

Comment:
Not the best comment i've never seen before :p
Maybe explain it's an upgrade of a ManyToMany who allow us to extend the default link ?